src/cclib/io/cjsonwriter.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, ccdata, terse=False, *args, **kwargs):
         Inputs:
           ccdata - An instance of ccData, parsed from a logfile.
         """
-
+        print ("TERSE:", terse)
```

Comment:
I'm not able to reproduce. I checked out your ```weird-init...``` brach, but I'm seeing ```TERSE: False``` with both Python 2.7.13 and 3.6.1 (both Mac Homebrew in virtual envs).